docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,240 @@
+--
+title: Running ASP.NET MVC applications in Windows Docker Containers
```

Comment:
Updated, I chose "migrating" over "containerizing". I want to avoid jargon to make it easier to localize later.